src/app/(main)/my-voyage/[teamId]/sprints/components/SprintActions.tsx

```diff
@@ -11,37 +11,57 @@ import Button from "@/components/Button";
 import routePaths from "@/utils/routePaths";
 
 interface SprintActionsProps {
-  teamId: string;
-  meetingId?: string;
-  sprintNumber: string;
+  params: {
+    teamId: string;
+    meetingId?: string;
+    sprintNumber: string;
+  };
 }
 
-export default function SprintActions({
-  teamId,
-  meetingId,
-  sprintNumber,
-}: SprintActionsProps) {
+export default function SprintActions({ params }: SprintActionsProps) {
+  const [teamId, meetingId, sprintNumber] = [
+    params.teamId,
+    params.meetingId,
+    params.sprintNumber,
+  ];
   const submitVoyageIsAllowed = sprintNumber === "5" || sprintNumber === "6";
   return (
-    <div className="flex justify-between p-5 border border-base-100 shadow-md bg-base-200 rounded-2xl">
+    <div className="flex justify-between p-5 border shadow-md border-base-100 bg-base-200 rounded-2xl">
       {/* TODO: add animated variant to Button.tsx ??? */}
-      <Link href={meetingId ? routePaths.submitVoyagePage(meetingId) : "/"}>
+      <Link
+        href={
+          meetingId
+            ? routePaths.submitVoyagePage(teamId, sprintNumber, meetingId)
+            : "/"
```

Comment:
is this supposed to be the dashboard page? If so, I'd rather you use that path instead

Reply:
same applies to the other links where you have this

Reply:
Done!